integration-testing/build.gradle

```diff
@@ -56,7 +56,6 @@ task npmTest(type: Test) {
 task mavenTest(type: Test) {
     def sourceSet = sourceSets.mavenTest
     dependsOn(project(':').getTasksByName("publishToMavenLocal", true))
-    dependsOn.remove(project(':').getTasksByName("dokka", true))
```

Comment:
Please elaborate on why did you remove this line

Reply:
No `remove` in 6.3 :(

Reply:
I do belive we still build all docs no CI now because the docs are needed for `knitCheck` task which we run in CI to ensure that we catch fast the cases where somebody committed updates to docs and forgot to run `knit` task.